Plugins/SQL/SQL.cpp

```diff
@@ -347,6 +347,7 @@ Events::ArgumentStack SQL::PreparedObjectFull(Events::ArgumentStack&& args)
 {
     auto position = Events::ExtractArgument<int32_t>(args);
     auto value = Events::ExtractArgument<API::Types::ObjectID>(args);
+    auto base64 = Events::ExtractArgument<int32_t>(args);
```

Comment:
Can you put this in a try/catch, and default to true if it catches something. That way we preserve both API and ABI compat.
We can remove the try/catch when we make a breaking change release.